File: baekjoon/dart/10871_X보다 작은 수.dart
void main(List<String> args) {
  List<String> input = ["10 5", "1 10 4 9 2 3 8 5 7 6"];
  List<String> one = input[0].split(" ");
  List<String> two = input[1].split(" ");

  List<int> a = one.map((e) => int.parse(e)).toList();
  List<int> b = two.map((e) => int.parse(e)).toList();

  List<int> arr = [];
  for (int i = 0; i < a[0]; i++) {
    if (b[i] < a[1]) {
      arr.add(b[i]);
    }
  }
  print(arr.join(" "));
}

File: baekjoon/javascript/10871 x보다 작은 수.js
// const input = require("fs").readFileSync("/dev/stdin").toString().trim().split('\n');
// const a = input[0].split(" ").map(n=>+n)
// const b = input[1].split(" ").map(n=>+n)

// let arr = [];
// for(let i = 0; i<a[0]; i++){
 
//     if(b[i]<a[1]){
//        arr.push(b[i])
//     }
   

// } console.log(arr.join(" "))

// 더 간결한 코드 

const input = require("fs").readFileSync("/dev/stdin").toString().trim().split(" ").map(n=>Math.pow(+n,2));
const a = input.reduce((a,b) => (a+b));
console.log(a%10);
